pysmartmeter/binary_sml_handler.py

```diff
@@ -0,0 +1,121 @@
+from crccheck.crc import Crc16X25
```

Comment:
Where is "crccheck" from?
dependencies should be defined in pyproject.toml and freezed by pip-tools calling: `./cli.py update`